src/main/java/lol/client/ai/RandomAITower.java

```diff
@@ -0,0 +1,60 @@
+package lol.client.ai;
+
+import java.util.*;
+import lol.game.*;
+import lol.game.action.*;
+
+public class RandomAITower extends AIBase {
```

Comment:
This class shares almost all its code with `RandomAI`. Please inherits from RandomAI, extract some methods (to attack nexus, and move) as protected methods, and use that in your `RandomAITower` to make it much smaller.